content/en/xtc-release-notes/v40.1.md

```diff
@@ -0,0 +1,26 @@
+---
+title: XTC v40.1
```

Comment:
XTC usually does not have minor versions, in this case there was one just for a fix. Please add the contents of v40.1 to the v40 page (same for other minor versions).